rotkehlchen/db/dbhandler.py

```diff
@@ -662,11 +662,19 @@ def remove_from_ignored_assets(self, write_cursor: 'DBCursor', asset: Asset) ->
             (asset.identifier,),
         )
 
-    def get_ignored_assets(self, cursor: 'DBCursor') -> List[Asset]:
-        cursor.execute(
-            'SELECT value FROM multisettings WHERE name="ignored_asset";',
-        )
+    def get_ignored_assets(self, cursor: 'DBCursor', only_nfts: bool = False) -> List[Asset]:
+        """
+        Retrieve all ignored assets.
+        If `only_nfts` is True, only ignored nfts are returned.
+        """
         assets = []
+        bindings = []
+        query = 'SELECT value FROM multisettings WHERE name="ignored_asset"'
```

Comment:
you need a space here. Since if `only_nfts` is True you got: `name="ignored_asset"AND value LIKE` ... which still works fine but only by chance since you end with a `"`

```suggestion
        query = 'SELECT value FROM multisettings WHERE name="ignored_asset" '
```

Reply:
okay